bindings/nodejs/src/logical_plan.rs

```diff
@@ -46,10 +46,11 @@ impl JsLogicalPlan {
         Ok(())
     }
 
-    #[napi(catch_unwind)]
-    pub async fn record_batches(&self) -> napi::Result<Vec<crate::record_batch::JsRecordBatch>> {
-        self.execute_inner().await?.record_batches().await
-    }
+    // TEMPORARILY DISABLED -- we need to figure out how to convert the raw record batches to JS objects.
+    // #[napi(catch_unwind)]
+    // pub async fn record_batches(&self) -> napi::Result<Vec<crate::record_batch::JsRecordBatch>> {
+    //     self.execute_inner().await?.record_batches().await
+    // }
```

Comment:
open ticket?

is it better to make this private and panic rather than carry around commented out code?

Reply:
I don't think it really warrants an issue yet. There hasn't been any requests for this specific feature, and knowing it is going to be quite challenging, maybe it's best to just remove all of this for now?